src/db/client.ts

```diff
@@ -0,0 +1,59 @@
+import pg, { QueryResultRow } from 'pg';
+
+class DBClient {
+    
+    private _pool: pg.Pool;
```

Comment:
We can remove the underscore from the variable name using the `private` keyword.